microprofile/oidc/src/main/java/io/helidon/microprofile/oidc/OidcCdiExtension.java

```diff
@@ -45,7 +45,7 @@ private void registerOidcSupport(@Observes @Initialized(ApplicationScoped.class)
             // only configure if security is enabled
             ServerCdiExtension server = bm.getExtension(ServerCdiExtension.class);
 
-            server.serverRoutingBuilder().register(OidcService.create(config));
+            OidcFeature.create(config).setup(server.serverRoutingBuilder());
```

Comment:
This is not correct. Please use `server.serverRutingBuilder.addFeature(OidcFeature.create(config))`

Reply:
Features have `Weight` and are ordered by routing to be executed in the correct order. See `docs-internal/nima-http-features.md` for details